prime-router/src/main/resources/metadata/fhir_transforms/common/suppress-device-information.yml

```diff
@@ -0,0 +1,28 @@
+## Suppresses Device information OBX-18-1,OBX-18-2,OBX-18-3
```

Comment:
These can all be suppressed by overiding element "obx-equipment-instance-identifier" in the mappings

  - name: obx-equipment-instance-identifier
    condition: false

Reply:
That does make it a lot easier. I was also trying to start using enrichment schemas for stuff like that, but this is a good example of how much more complicated transforms are in FHIR to FHIR.